### PR TITLE
fix: Preview image are not displayed in activity -EXO-60824

### PR DESCRIPTION
**Pull request by akhanfir**
Prior to this change, when move image of an activity from Activity Stream Documents to another folder, the image of this activity does not show up in the activities page After this change, the moved image is displayed in the activities page.

### CODE REVIEW

**Review by ahamdi — CHANGES_REQUESTED** [review]
Could you add a Unit test for the new function ?

File: pom.xml [hunk 1]
```diff
@@ -40,10 +40,10 @@
     <!-- **************************************** -->
     <!-- Project Dependencies                     -->
     <!-- **************************************** -->
-    <org.exoplatform.social.version>6.5.x-maintenance-SNAPSHOT</org.exoplatform.social.version>
+    <org.exoplatform.social.version>6.5.x-exo-SNAPSHOT</org.exoplatform.social.version>
     <addon.exo.jcr.version>6.5.x-maintenance-SNAPSHOT</addon.exo.jcr.version>
     <addon.exo.tasks.version>3.5.x-maintenance-SNAPSHOT</addon.exo.tasks.version>
-    <org.exoplatform.platform-ui.version>6.5.x-maintenance-SNAPSHOT</org.exoplatform.platform-ui.version>
+    <org.exoplatform.platform-ui.version>6.5.x-exo-SNAPSHOT</org.exoplatform.platform-ui.version>
```
Comment (azayati) [inline pom.xml]: this should not be changed, the branch should be rebased

**Review by azayati — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -40,10 +40,10 @@
     <!-- **************************************** -->
     <!-- Project Dependencies                     -->
     <!-- **************************************** -->
-    <org.exoplatform.social.version>6.5.x-maintenance-SNAPSHOT</org.exoplatform.social.version>
+    <org.exoplatform.social.version>6.5.x-exo-SNAPSHOT</org.exoplatform.social.version>
```
Comment (ahamdi) [inline pom.xml]: @akhanfir Could you revert this change ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

**Review by ahamdi — APPROVED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**